crates/ruff_linter/resources/test/fixtures/flake8_pyi/PYI064.py

```diff
@@ -0,0 +1,21 @@
+from typing import Final, Literal
+
+x: Final[Literal[True]] = True  # PYI064
+y: Final[Literal[None]] = None  # PYI064
+z: Final[Literal[  # PYI064
+    "this is a really long literal, that won't be rendered in the issue text"
+]] = "this is a really long literal, that won't be rendered in the issue text"
+
+# This should be fixable, and marked as safe
+w1: Final[Literal[123]]  # PYI064
+
+# This should not be fixable
+w2: Final[Literal[123]] = "random value"  # PYI064
+
+n1: Final[Literal[True, False]] = True # No issue here
+n2: Literal[True] = True  # No issue here
+
+PlatformName = Literal["linux", "macos", "windows"]
+PLATFORMS: Final[set[PlatformName]] = {"linux", "macos", "windows"}  # No issue here
+
+foo: Final[{1, 2, 3}] = {1, 2, 3}  # No issue here
```

Comment:
Right now this mimics `flake8-pyi` behaviour, however I think raising the issue here should be fine?